python/paddle/trainer_config_helpers/layers.py

```diff
@@ -1454,6 +1472,19 @@ def first_seq(input,
 
 
 class ExpandLevel(object):
+    """
+    Please refer to AggregateLevel first.
+
+    ExpandLevel supports two modes:
+
+    - :code:`ExpandLevel.FROM_TIMESTEP` means the expandation acts on each
+      timestep of a sequence, :code:`NO_SEQUENCE` will be expanded to
+      :code:`SEQUENCE` or :code:`SUB_SEQUENCE`.
```

Comment:
"acts on each timestep of a sequence" is not correct because the input for FROM_TIMESTEP is non-sequence. Perhaps we should consider change the name to FROM_NON_SEQUENCE.